Source/CesiumRuntime/Public/CesiumMetadataValueType.h

```diff
@@ -106,10 +106,72 @@ enum class ECesiumMetadataType : uint8 {
   Mat3 = int(CesiumGltf::PropertyType::Mat3),
   Mat4 = int(CesiumGltf::PropertyType::Mat4),
   Boolean = int(CesiumGltf::PropertyType::Boolean),
-  Enum = int(CesiumGltf::PropertyType::Enum),
   String = int(CesiumGltf::PropertyType::String),
```

Comment:
I don't understand why `Enum` was removed. Since this is supposed to represent the "true" type from the extension, it's important that the Enum type be represented. I can see that `TypeToMetadataValueType` can't distinguish Scalar from Enum as written, but that can be solved by passing in an extra parameter. It looks like every place that `TypeToMetadataValueType` is called, there's also an `EnumDefinition` available.